program/convert-miles-to-meters/ConvertMilesToMeters.kt

```diff
@@ -0,0 +1,16 @@
+import java.util.Scanner
+
+fun main() {
+    print("Enter Miles : ")
+    println()
+    var miles = readLine()
+
+    println("$miles miles in meters : " + ConvertMilesToMeters(miles!!.toInt()))
+
+}
+
+private fun ConvertMilesToMeters(miles: Int): Double {
```

Comment:
```suggestion
private fun ConvertMilesToMeters(miles: Double): Double {
```
I think `miles` can also be present in `double`.